presto-main/src/main/java/com/facebook/presto/sql/planner/SubqueryPlanner.java

```diff
@@ -291,6 +292,33 @@ private PlanBuilder appendQuantifiedComparisonApplyNode(PlanBuilder subPlan, Qua
             // given subquery is already appended
             return subPlan;
         }
+        switch (quantifiedComparison.getComparisonType()) {
+            case EQUAL:
+                switch (quantifiedComparison.getQuantifier()) {
```

Comment:
Well, nested switches with 'break's in between may look a bit weird.
However, I've tried using 'if's with 'isOneOf' as an experiment right now and it seems like it doesn't add much clarity.
I think the main issue impacting readability is that some cases are long. I've extracted 2 of them into their own methods, so overall switch looks better now.